### DISCUSSION TITLE
config help

### DISCUSSION THREAD

**Discussion by cupidvincent** [renovatebot/renovate#13774] [Request Help] [answered]
i got confused by the documentation from the start, is there any videos on how to start it with gitlab?

**Comment by viceice** [comment]
Tryout our ofiicial gitlab pipeline templates at https://gitlab.com/renovate-bot/renovate-runner

**Reply by cupidvincent** [reply]
thanks so much ill take a look on it  👍

**Comment by rarkins** [comment, marked as answer]
If you google for blog posts about renovate bot and gitlab, you can also find quite a few examples: https://www.google.com/search?q=blog+renovate+bot+gitlab

**Reply by cupidvincent** [reply]
thanks so much ill take a look on it  👍
